i18n/en-US.yml

```diff
@@ -767,6 +767,12 @@ common:
     calories: "{calories, number} Cal"
     noItineraryToDisplay: No itinerary to display.
     transfers: "{transfers, plural, =0 {} one {# transfer} other {# transfers}}"
+    # This string will have a formatted number before it, including units.
+    relativeCo2: >
+     {isMore, select,
+      true {more}
+      other {less}
+      } CO2 than driving alone.
```

Comment:
I would shorten this string so that the time isn't pushed awkwardly to the middle. Maybe `XX% of car emissions`? Either that or we should figure out a way to force a line break

Reply:
We should also be careful with language. if a car emits 100 grams, and we say 85% less co2 then isn't that 15 grams? is that what we mean or do we mean 85 grams

Reply:
I think having the word "less" and "more" is pretty important to make it clear to understand at a glance.

Reply:
OK I aligned all the attributes to the right. Pretty big change, but I think I do like it more. Let me know what you think.

Reply:
This definitely looks better thanks! Feels a bit busy but we can probably leave that for the redesign